starters/gatsby-starter-blog-theme/.prettierrc

```diff
@@ -1,7 +1,4 @@
 {
-  "endOfLine": "lf",
```

Comment:
I don't think the changes here (on all files under `starters/gatsby-starter-blog-theme/*`) are necessary.

Could you please revert the changes in these files so that the starter remains as it was? It keeps the history of the files cleaner if we reduce the churn of unnecessary changes. 🙂